### ISSUE TITLE
Installed server can't find html files

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#139]
Opening the web page at / using `server_dev.py` works fine, but opening it with the installed version of the server does not since it's searching for the html files in a place where they haven't been installed.

```
$ ga4gh_server wormtable ga4gh-example-data
 * Running on http://0.0.0.0:8000/ (Press CTRL+C to quit)
 * Restarting with stat
Traceback (most recent call last):
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "build/bdist.macosx-10.10-intel/egg/ga4gh/frontend.py", line 117, in index
    'index.html', info=app.serverStatus.getStatusInfo())
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/flask/templating.py", line 127, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/jinja2/environment.py", line 791, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/jinja2/environment.py", line 765, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/flask/templating.py", line 60, in get_source
    return loader.get_source(environment, local_name)
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/jinja2/loaders.py", line 163, in get_source
    f = open_if_exists(filename)
  File "/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/jinja2/utils.py", line 150, in open_if_exists
    return open(filename, mode)
IOError: [Errno 20] Not a directory: u'/Users/dannyc/ga4gh/new/lib/python2.7/site-packages/ga4gh-0.1.0a1-py2.7.egg/ga4gh/templates/index.html'
```

**Comment by Naburimannu** [comment]
Could not reproduce.
(Was this triggered immediately, or is it a response to some client action?)

```
$ ga4gh_server wormtable ga4gh-example-data
 * Running on http://0.0.0.0:8000/
 * Restarting with reloader
```

I'm getting various client errors, so it could be a problem on my end...

**Comment by dcolligan** [comment]
You may have an unzipped egg in your site-packages for some reason?  Try blowing any ga4gh packages away and reinstalling and then trying to reproduce.

**Comment by dcolligan** [comment]
Fixed in #149
